### PR TITLE
feat: Make suggestion popup full width (WEBAPP-5517)

### PR DESCRIPTION
**Pull request by AndyLnd**
This introduces simplebar (not dead) as an alternative to antiscroll (dead). It is used for now on the suggestions popup and achieves what antiscroll couldn't: allow me to make it full width.
Simplebar will be explored further in coming PRs and eventually replace antiscroll everywhere.

### CODE REVIEW

File: app/script/components/mentionSuggestions.js [hunk 1]
```diff
@@ -56,7 +56,7 @@ z.components.MentionSuggestions = class MentionSuggestions {
     });
 
     this.shouldUpdateScrollbar = ko.pureComputed(() => this.suggestions()).extend({notify: 'always', rateLimit: 100});
-    this.shouldUpdateScrollbar.subscribe(() => this.setWrapperSize(), null, 'beforeChange');
+    /*this.shouldUpdateScrollbar.subscribe(() => this.setWrapperSize(), null, 'beforeChange');
```
Comment (gregor) [inline app/script/components/mentionSuggestions.js]: Let's remove code that has been commented out

File: app/script/components/mentionSuggestions.js [hunk 2]
```diff
@@ -117,24 +117,26 @@ z.components.MentionSuggestions = class MentionSuggestions {
   }
 
   updateScrollPosition(selectedNumber) {
-    const suggestionList = document.querySelector('.mention-suggestion-list');
-    if (!suggestionList) {
+    const scrollContent = document.querySelector(
+      '.conversation-input-bar-mention-suggestion > .simplebar-scroll-content'
```
Comment (gregor) [inline app/script/components/mentionSuggestions.js]: Put in const if too long for single line: 3 => 2

File: app/style/common/mixins.less [hunk 3]
```diff
@@ -242,10 +242,6 @@
   visibility: hidden;
 }
 
-.visible {
-  visibility: visible;
-}
-
```
Comment (gregor) [inline app/style/common/mixins.less:248]: Why removing a mixin?

File: package.json [hunk 4]
```diff
@@ -60,6 +60,7 @@
     "redux-logger": "3.0.6",
     "redux-thunk": "2.3.0",
     "sdp-transform": "2.4.1",
+    "simplebar": "3.0.0-beta.5",
```
Comment (gregor) [inline package.json:63]: Beta?

File: app/style/common/mixins.less (hunk 3 above)
Comment (AndyLnd) [reply app/style/common/mixins.less:248]: It is not used anywhere and actually clashes with simplebar.

File: app/script/components/mentionSuggestions.js [hunk 5]
```diff
@@ -117,24 +107,25 @@ z.components.MentionSuggestions = class MentionSuggestions {
   }
 
   updateScrollPosition(selectedNumber) {
-    const suggestionList = document.querySelector('.mention-suggestion-list');
-    if (!suggestionList) {
+    const scrollContentSelector = '.conversation-input-bar-mention-suggestion > .simplebar-scroll-content';
```
Comment (atomrc) [inline app/script/components/mentionSuggestions.js]: Let's try not to depend on classes we don't manage ourselves. Here we create an implicit dependency to `simplebar`.
There might be some cleaner way to get the scroll height

File: package.json (hunk 4 above)
Comment (AndyLnd) [reply package.json:63]: Yes, this has performance improvements over the latest stable version.
https://github.com/Grsmto/simplebar/releases/tag/simplebar%403.0.0-beta.0

File: app/script/components/mentionSuggestions.js [hunk 6]
```diff
@@ -56,16 +57,6 @@ z.components.MentionSuggestions = class MentionSuggestions {
     });
 
     this.shouldUpdateScrollbar = ko.pureComputed(() => this.suggestions()).extend({notify: 'always', rateLimit: 100});
```
Comment (atomrc) [inline app/script/components/mentionSuggestions.js]: Can you try `ko.pureComputed(this.suggestions)./*...*/`?

**Pull request merged**